capstone/src/main/java/com/google/univiz/api/MapsData.java

```diff
@@ -1,4 +1,42 @@
 package com.google.univiz.api;
 
-/** The MapsData class will contain all college data that is specific to the Maps feature */
-public class MapsData {}
+import com.google.auto.value.AutoValue;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+
+/**
+ * MapsData is a class representing all relevant college data that is specific to the Maps feature.
+ */
+@GenerateTypeAdapter
+@AutoValue
+public abstract class MapsData {
+
+  public abstract String name();
+
+  public abstract String city();
+
+  public abstract boolean isMainCampus();
+
+  public abstract double latitude();
+
+  public abstract double longitude();
+
+  public static Builder builder() {
+    return new AutoValue_MapsData.Builder();
+  }
+
+  @AutoValue.Builder
+  public abstract static class Builder {
```

Comment:
I don't think this `Builder` class (and all its `set` methods) need to be public. package-private should be enough.